lib/debootstrap.sh

```diff
@@ -616,6 +616,21 @@ prepare_partitions()
         fi
 	fi
 
+	if [[ $SRC_EXTLINUX == yes ]]; then
+		mkdir -p $SDCARD/boot/extlinux
+
+		cat << EOF > "$SDCARD/boot/extlinux/extlinux.conf"
```

Comment:
Should we rather make extlinux.conf file in https://github.com/armbian/build/tree/master/config/bootenv and adjust values here.